code/datums/diseases/rhumba_beat.dm

```diff
@@ -10,33 +10,33 @@
 	permeability_mod = 1
 	severity = DISEASE_SEVERITY_BIOHAZARD
 
-/datum/disease/rhumba_beat/stage_act()
+/datum/disease/rhumba_beat/stage_act(delta_time, times_fired)
 	. = ..()
 	if(!.)
 		return
 
 	switch(stage)
 		if(2)
-			if(prob(45))
+			if(DT_PROB(26, delta_time))
 				affected_mob.adjustFireLoss(5, FALSE)
-			if(prob(1))
+			if(DT_PROB(0.5, delta_time))
 				to_chat(affected_mob, "<span class='danger'>You feel strange...</span>")
 		if(3)
-			if(prob(5))
+			if(DT_PROB(2.5, delta_time))
 				to_chat(affected_mob, "<span class='danger'>You feel the urge to dance...</span>")
-			else if(prob(5))
+			else if(DT_PROB(2.5, delta_time))
 				affected_mob.emote("gasp")
-			else if(prob(10))
+			else if(DT_PROB(5, delta_time))
 				to_chat(affected_mob, "<span class='danger'>You feel the need to chick chicky boom...</span>")
 		if(4)
-			if(prob(20))
-				if (prob(50))
+			if(DT_PROB(10, delta_time))
+				if(prob(50))
```

Comment:
Missed one?

Reply:
It's inside of the `DT_PROB` if I also changed it I'd halve the effective rate.